src/components/designSystem/Dialog.tsx

```diff
@@ -13,34 +13,37 @@ import { useEffect } from 'react'
 import { theme, ButtonGroup } from '~/styles'
 
 import { Typography } from './Typography'
+import { Icon, IconName } from './Icon'
```

Comment:
I'm wondering if this is something we will reuse or if we're changing a base component just to fit an edge case ?

Reply:
Not sure how to nicely handle that tbh if I don't update the dialog api.

Reply:
Seen with product, icon is removed for now, we'll see later if we want to use icon on modals